mods/ra/maps/allies-03a/allies03a.lua

```diff
@@ -107,11 +107,10 @@ InitPlayers = function()
 end
 
 AddObjectives = function()
-	KillBridges = player.AddObjective("Destroy all bridges.")
-	TanyaSurvive = player.AddObjective("Tanya must survive.")
-	KillUSSR = player.AddObjective("Destroy all Soviet oil pumps.", "Secondary", false)
-	FreePrisoners = player.AddObjective("Free all Allied soldiers and keep them alive.", "Secondary", false)
-	ussr.AddObjective("Bridges must not be destroyed.")
```

Comment:
This objective isn't added back

Reply:
Because it is for the enemy and unused. The ID isn't even saved.